BudgetSquirrel.Business/BudgetSquirrel.Business.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
Why not upgrade this to be 3.1 or at least 2.1

Not that it matters a lot but Microsoft has LTS for 2.1 and 3.1: https://dotnet.microsoft.com/download/dotnet-core